cnf-certification-test/faredge/suite.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2020-2023 Red Hat, Inc.
```

Comment:
I would prefer a more generic name for the suite such as "performance" or similar. Faredge are not the only CNFs that could use high performance/low latency configurations